admin/jobs/river/river.go

```diff
@@ -105,6 +106,7 @@ func New(ctx context.Context, dsn string, adm *admin.Service) (jobs.Client, erro
 		newPeriodicJob(&TrialEndCheckArgs{}, "10 1 * * *", true),                 // daily at 1:10am UTC
 		newPeriodicJob(&TrialGracePeriodCheckArgs{}, "15 1 * * *", true),         // daily at 1:15am UTC
 		newPeriodicJob(&SubscriptionCancellationCheckArgs{}, "20 1 * * *", true), // daily at 1:20am UTC
+		newPeriodicJob(&HibernateInactiveOrgsArgs{}, "0 1 * * 0", true),          // weekly at 1am UTC on Sunday
```

Comment:
Can you make this 7am UTC on Monday instead? We don't want things happening automatically in the weekend if we can avoid it